tendrl/node_agent/node_sync/disk_sync.py

```diff
@@ -79,9 +78,21 @@ def get_node_disks():
                         else:
                             rv['used_disks_id'].append(disk['disk_id'])
         else:
-            LOG.error(err)
+            Event(
+                Message(
+                    priority="error",
+                    publisher=tendrl_ns.publisher_id,
+                    payload={"message": err}
+                )
+            )
     else:
-        LOG.error(err)
+        Event(
+            Message(
+                priority="error",
+                publisher=tendrl_ns.publisher_id,
+                payload={"message": err}
+            )
+        )
```

Comment:
In exception class you can use exception object, but now you can't use exception object it gives serialization problem so make sure this "ex" is string or not